src/LiveComponent/doc/index.rst

```diff
@@ -3829,6 +3834,44 @@ uses Symfony's test client to render and make requests to your components::
     The ``InteractsWithLiveComponents`` trait can only be used in tests that extend
     ``Symfony\Bundle\FrameworkBundle\Test\KernelTestCase``.
 
+When testing live components, you can assert whether a specific event has been emitted using new assertion methods provided by the ``InteractsWithLiveComponents`` trait.
+
+Use the following assertions to verify event emissions from your live component::
+
+    use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+    use Symfony\UX\LiveComponent\Test\InteractsWithLiveComponents;
+
+    class MyComponentTest extends KernelTestCase
+    {
+        use InteractsWithLiveComponents;
+
+        public function testEventEmission(): void
+        {
+            $testComponent = $this->createLiveComponent(
+                name: 'MyComponent',
+                data: ['foo' => 'bar'],
+            );
+
+            // Call an action that emits an event
+            $render = $testComponent->call('triggerEvent')->render();
+
+            // Assert that the event was emitted
+            $this->componentHasEmittedEvent($render, 'event-name', [
+                'eventArg1' => 'value',
+            ]);
+
+            // Assert that a specific event was *not* emitted
+            $this->assertComponentNotEmitEvent($render, 'unwanted-event');
+        }
+    }
+
+.. note::
+
+    These assertions are only available when using the ``InteractsWithLiveComponents`` trait in a test class that extends
+    ``Symfony\Bundle\FrameworkBundle\Test\KernelTestCase``.
+
+---
```

Comment:
I don't think this whole block is useful as its own, IMHO it could be merged with the previous code example. 

By doing that, we won't repeat twice "When testing live components, you can assert (...) provided by the ``InteractsWithLiveComponents`` trait" nor "These assertions are only available when using (...)".